### ISSUE TITLE
Proposal: init and global blocks

### ISSUE THREAD

**Issue by andronat** [daniel-grumberg/varan-div#17]
Sometimes it is useful to do some kind of setup before any execution starts. e.g. defaulting some global variables, setuping ignored filepaths, etc.

My suggestion is something like this:
```
include "event.h"
include "assert.h"

init { ...<C code>... }

nothing => fdatasync(fd)
...
```

**Comment by luisggpina** [comment]
Should we also add to this the idea of allowing arbitrary C code above a delimiter in the file, such as a line with just `---`?

**Comment by andronat** [comment]
Just for the record, we discussed that together with the `init` block, we will also support the `global` block which would be for defining global variables or arbitrary C functions.

**Comment by daniel-grumberg** [comment]
I'll have a PR up for it in a sec

**Comment by luisggpina** [comment]
Fixed by #26
